src/mrpro/data/traj_calculators/_KTrajectoryPulseq.py

```diff
@@ -58,28 +61,35 @@ def __call__(self, kheader: KHeader) -> KTrajectoryRawShape:
 
         # calculate k-space trajectory using PyPulseq
         k_traj_adc_numpy, _, _, _, _ = seq.calculate_kspace()
-        k_traj_adc = torch.from_numpy(k_traj_adc_numpy)
+        k_traj_adc = torch.tensor(k_traj_adc_numpy, dtype=torch.float32)
 
         unique_idxs = {label: torch.unique(getattr(kheader.acq_info.idx, label)) for label in ['k1', 'k2']}
 
-        k1 = len(unique_idxs['k1'])
-        k2 = len(unique_idxs['k2'])
+        n_k1 = len(unique_idxs['k1'])
+        n_k2 = len(unique_idxs['k2'])
 
         n_samples = kheader.acq_info.number_of_samples
         n_samples = torch.unique(n_samples)
         if len(n_samples) > 1:
             raise ValueError('We currently only support constant number of samples')
-        k0 = int(n_samples.item())
+        n_k0 = int(n_samples.item())
 
         sample_size = n_samples.shape[0]
```

Comment:
n_other ?

Reply:
somethink looks wrong here.

Have we tested this with multiple others?

Either n_samples is a single value, then n_k0 = int(n_samples.item()) works
OR
  sample_size = n_samples.shape[0] works.

Reply:
if this indeed works, can you add a comment how and why?

Reply:
good point, this should not work for n_other > 1. And obviously it was never tested before, because this is in the code for quite some time. 

How should we do it? 
```
  n_samples_unique = torch.unique(n_samples)
  if len(n_samples_unique) > 1:
      raise ValueError('We currently only support constant number of samples')
  n_k0 = int(n_samples_unique.item())
```

and than just `n_other = n_samples.shape[0]`?
This would at least work for other > 1 with all having the same `n_k0`, no ?!

I am not sure how `n_samples` looks like for multiple `other`.
We should get some test data for this... maybe we can acquire some on Tuesday @JoHa0811

Reply:
Actually not needed at all ... at least at the moment. See other comment by @fzimmermann89 and my answer.